lib/ship.rb

```diff
@@ -10,15 +10,11 @@ def initialize(name, length)
   end
 
   def hit
-      @health -= 1
+    @health -= 1
   end
 
   def sunk?
-    if @health == 0
-      true
-    else
```

Comment:
Is this else to prevent a nil?

Reply:
Actually, on line 17, I put it back to @health == 0.  I probably did change that to prevent a nil, but honestly even with the change I don't believe I accounted for the nil.  @health is perfectly within reach of the ship scope.

Reply:
Gotcha. Thanks for clarifying!